waffle_hub/experimental/auto_label/extract_frames.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import os
+from pathlib import Path
+from typing import Union
+
+import cv2
+from tqdm import tqdm
+from waffle_utils.file import io, search
+
+
+def extract_frames(video_path: Union[str, Path], output_dir: Union[str, Path], interval: int = 1):
+    video_path = Path(video_path)
+
+    cap = cv2.VideoCapture(str(video_path))
+
+    frame_dir = Path(output_dir, video_path.stem)
+    io.make_directory(frame_dir)
+
+    frame_id = 0
+    while cap.isOpened():
+        ret, frame = cap.read()
+        if not ret:
+            break
+        if frame_id % interval == 0:
+            frame_path = frame_dir / f"{frame_id:06d}.jpg"
+            cv2.imwrite(str(frame_path), frame)
```

Comment:
Need to use waffle_utils, Because, Korean in the path can cause unexpected errors.